### PR TITLE
feat: Last Modified Header + sdkConfig - FDN-282

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs [hunk 1]
```diff
@@ -46,7 +48,7 @@ public class EnvironmentConfigManager : IDisposable
             LocalBucketing localBucketing,
             EventHandler<DevCycleEventArgs> initializedHandler = null,
             DevCycleRestClientOptions restClientOptions = null
-        )
+            )
```
Comment (jonathannorris) [inline DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:51]: should this be tabbed?

Comment (JamieSinn) [reply DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs:51]: Auto format, so yes

File: DevCycle.SDK.Server.Local/Api/EventQueue.cs [hunk 2]
```diff
@@ -153,24 +157,71 @@ public virtual void QueueEvent(DevCyclePopulatedUser user, DevCycleEvent @event,
...
         }
 
+        public virtual void QueueSDKConfigEvent(RestRequest request, RestResponse response)
+        {
+            var popU = new DevCyclePopulatedUser(
+                new DevCycleUser(userId: $"{localBucketing.ClientUUID}@{Dns.GetHostName()}"));
+            QueueEvent(popU, new DevCycleEvent(
+                type: EventTypes.sdkConfig,
+                target: request.Resource,
+                value: -1,
```
Comment (jonathannorris) [inline DevCycle.SDK.Server.Local/Api/EventQueue.cs:176]: any way to get the response time in ms here?

Comment (JamieSinn) [reply DevCycle.SDK.Server.Local/Api/EventQueue.cs:176]: Not without wrapping our own timer - which isn't recommended

**Review by jonathannorris — APPROVED** [review]

**Pull request merged**